src/main.ts

```diff
@@ -0,0 +1,99 @@
+import * as core from "@actions/core"
+import * as exec from "@actions/exec"
+import * as github from "@actions/github"
+import { GitHub } from "@actions/github/lib/utils"
+import * as tc from "@actions/tool-cache"
+import * as fs from "fs"
```

Comment:
Same as before. It helps legibility to know what you import, IMO.

Reply:
See #8 for follow up